extensions/sftp/tests/FetchSFTPTests.cpp

```diff
@@ -74,18 +72,15 @@ class FetchSFTPTestsFixture {
     LogTestController::getInstance().setDebug<processors::LogAttribute>();
     LogTestController::getInstance().setDebug<SFTPTestServer>();
 
-    // Create temporary directories
-    testController.createTempDirectory(src_dir);
-    REQUIRE(src_dir != nullptr);
-    testController.createTempDirectory(dst_dir);
-    REQUIRE(dst_dir != nullptr);
+    REQUIRE_FALSE(src_dir.empty());
+    REQUIRE_FALSE(dst_dir.empty());
+    REQUIRE(plan);
```

Comment:
What is the purpose of the last REQUIRE statement?

Reply:
To make sure that the pointer is not null, since we dereference it later.  An assertion failure here is better than a segmentation fault later.

Reply:
Got it. Thanks :)